spec/classes/config_spec.rb

```diff
@@ -114,7 +125,7 @@
                 debug = err
                 syslog = no
                 foreground = yes
-                pid =
+                pid = /var/run/stunnel/stunnel.pid
```

Comment:
There shouldn't be a pid on systemd

Reply:
@jeefberkey If there isn't, you'll never log into your EL7 systems again. This is actually why this PR exists.

Reply:
What? systemd takes care of the problems pidfiles were created to solve